source/renderer/app/stores/ProfileStore.js

```diff
@@ -34,7 +34,8 @@ export default class SettingsStore extends Store {
   ];
 
   @observable systemLocale: string = LOCALES.english;
-  @observable bigNumberDecimalFormat = {
+  @observable
```

Comment:
@yakovkaravelov please remove the line break after the @observable / @computed / @action in this file (multiple places!)